hand.py

```diff
@@ -0,0 +1,41 @@
+"""a class for storing poker hands"""
+
+from deck import Deck
+from card import Card
+
+class Hand:
+    def __init__(self, hand_list):
+        self.hand_list = hand_list
```

Comment:
Similar to the previous comment. I would call this `cards`. We already know it's in a hand by the structure and `list` is not very descriptive of the contents.